.github/workflows/deploy.yml

```diff
@@ -1,12 +1,12 @@
-name: CI
+name: Deploy
 
 on:
   push:
     branches: [main]
 
 jobs:
   # Permissions to request and use OIDC tokens and access repository contents
-  deploy:
+  copilot:
```

Comment:
Can this bit only be tested upon merging?

Reply:
This is setup to go to "test" so I would argue that this _is_ the test. 😀